installer/manifests/keptn/charts/control-plane/templates/api-gateway-nginx.yaml

```diff
@@ -77,7 +77,7 @@ data:
       proxy_set_header X-Forwarded-Proto $scheme;
     }
 
-    location ~* {{ .Values.prefixPath }}/bridge/.* {
+    location ~* {{ .Values.prefixPath }}/bridge {
```

Comment:
are you sure this works? if it's a direct regex matcher then i would assume that with this, no requests like `/bridge/somethingsomething` are now served correctly

Reply:
AFAIK nginx has a greedy regex parser. Since it matches the beginning of the URL it accepts it. We manually tested and it was working.